tests/fetch/example_idb_store.cpp

```diff
@@ -0,0 +1,40 @@
+#include <string.h>
+#include <stdio.h>
+#include <stdlib.h>
+#include <time.h>
+#include <emscripten/fetch.h>
+
+void success(emscripten_fetch_t *fetch)
+{
+  printf("IDB store succeeded.\n");
+  emscripten_fetch_close(fetch);
+}
+
+void failure(emscripten_fetch_t *fetch)
+{
+  printf("IDB store failed.\n");
+  emscripten_fetch_close(fetch);
+}
+
+void persistFileToIndexedDB(const char *outputFilename, uint8_t *data, size_t numBytes)
+{
+  emscripten_fetch_attr_t attr;
+  emscripten_fetch_attr_init(&attr);
+  strcpy(attr.requestMethod, "EM_IDB_STORE");
+  attr.attributes = EMSCRIPTEN_FETCH_REPLACE | EMSCRIPTEN_FETCH_PERSIST_FILE;
+  attr.requestData = (char *)data;
```

Comment:
Yeah, the implementation of `emscripten_fetch_close()` was partial, this is now implemented, and also documented more explicitly.